### PR TITLE
Dev containers & GHA Release Flow

### PR DESCRIPTION
**Pull request by pSchlarb**

### CODE REVIEW

File: .github/workflows/PR.yaml
Comment (pSchlarb) [inline .github/workflows/PR.yaml]: Needs to be changed to the correct branch after https://github.com/hyperledger/indy-test-automation/pull/120/ is merged

File: updateVersion.py
Comment (WadeBarnes) [inline updateVersion.py]:
```suggestion
print("Updated version of sovrin metadata.json to: ", version)
```

File: updateVersion.py
Comment (WadeBarnes) [inline updateVersion.py]: Would be an idea to extract `sovrin/metadata.json` as a constant so it's defined once.

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
Some small changes.  We also need to wait until the associate indy-test-automation PR is merged and we tag a version so we can reference the specific version in these workflows.

**Comment by WadeBarnes** [comment]
Added a ticket for future enhancements; https://github.com/sovrin-foundation/sovrin/issues/312

**Review by WadeBarnes — COMMENTED** [review]
Latest changes look good.  Should we be targeting the `stable` branch with these changes though?  The `stable` branch is traditionally used for releases.

**Review by WadeBarnes — APPROVED** [review]
Looks good.  The build failures are expected due to the permissions on PRs.  The DinD image is not being pushed to the repo's image repository.  It's passing because it's not trying to push the image.  The client build is failing because the image does not exist yet (expected, will be there once PR is merged).

Just waiting to see the results on https://github.com/pSchlarb/sovrin/actions/runs/3894517829

**Pull request merged**
